benchmark_test.go

```diff
@@ -218,3 +220,88 @@ func TestBenchmarkStatsPerPeer(t *testing.T) {
 	})
 	assert.Equal(t, want, statsServer.Aggregated(), "unexpected stats")
 }
+
+func TestBenchmarkOutput(t *testing.T) {
+	// Testing both plaintext and JSON output
+	tests := []struct {
+		name          string
+		opts          BenchmarkOptions
```

Comment:
do you need the entire `BenchmarkOptions` in the input, or you can just use `format string` here, and use it to construct the option in your test logic